test-prow-e2e.sh

```diff
@@ -28,3 +30,9 @@ oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
 oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
 
 CHROME_VERSION=$(google-chrome --version) ./test-gui.sh "${1:-e2e}"
+
+# Don't run Cypress tests if a specific Protractor scenario was passed as an argument.
+if [ $# -eq 0 ]; then
```

Comment:
For now, we can probably just run the Cypress tests if the `release` scenario was passed as an argument. Note that this means any tests added to Cypress will automatically be added to our release-informing jobs.

```suggestion
if [ $# -eq 0 || "$1" == "release" ]; then
```

Reply:
fixed and commented -thanks